cmd/minikube/cmd/start.go

```diff
@@ -126,6 +124,7 @@ func runStart(cmd *cobra.Command, args []string) {
 
 	var host *host.Host
 	start := func() (err error) {
+		fmt.Printf("Starting local OpenShift instance using '%s' hypervisor...\n", config.VMDriver)
```

Comment:
Ya, I am not able to think what approach to use to write any test case. Let me know @hferentschik , if you have any approach.